tests/test_basic.py

```diff
@@ -7,7 +7,7 @@
 
 class BasicTest(unittest.TestCase):
 
-    def test_pass(self):
+    def test_train_model(self):
```

Comment:
Can you rename this `test_pass`? Or remove the file altogether, as it was only meant as an example.